codegen/smithy-go-codegen/src/test/java/software/amazon/smithy/go/codegen/GenerateStandaloneGoModuleTest.java

```diff
@@ -25,8 +25,9 @@ public void testGenerateGoModule() throws Exception {
         var testPath = Files.createTempDirectory(getClass().getName());
         LOGGER.warning("generating test suites into " + testPath);
 
+        // symbol provider not needed here since only useFileWriter is called
         var fileManifest = FileManifest.create(testPath);
-        var writers = new GoWriterDelegator(fileManifest);
+        var writers = new GoDelegator(fileManifest, null);
```

Comment:
Can we create another constructor without a Symbol provider instead?

Reply:
I would err on the side of no since it's not really supposed to be used that way (useShapeWriter calls would explode). This is an ancient test and it was easier to just skip it instead of building up the GoSettings component required for the symbol provider.